tungsten_ci_utils/mirror_maven_repo/deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+repoUrl='http://ci-nexus.englab.juniper.net/repository/vmware-releases/'
```

Comment:
Make this configurable (config file or cli parameter)

Reply:
Done.